pkg/pillar/types/zedroutertypes.go

```diff
@@ -446,17 +486,35 @@ type DeviceNetworkStatus struct {
 	Ports   []NetworkPortStatus
 }
 
-func (status *DeviceNetworkStatus) GetPortByIfName(
-	ifname string) *NetworkPortStatus {
-	for _, portStatus := range status.Ports {
-		if portStatus.IfName == ifname {
-			log.Infof("Found NetworkPortStatus for %s", ifname)
+// GetPortByIfName - Get Port Status for port with given Ifname
+func (statusPtr *DeviceNetworkStatus) GetPortByIfName(
+	port string) *NetworkPortStatus {
```

Comment:
Why are you renaming the ifname argument back to port? I just fixed those to have names which mean something.

Reply:
good catch.. Copy paste error..